### PR TITLE
fix/7518

### LINKED ISSUE #7518: v12.2: online/offline status doesn't seem to work as expected
**Describe the bug**
When PF received a RADIUS Accounting-Request (Start) for a node, I can see following data in DB:

```sql
MariaDB [pf]> select * from node_current_session;
+-------------------+---------------------+----------------------+-----------+
| mac               | updated             | last_session_id      | is_online |
+-------------------+---------------------+----------------------+-----------+
| 00:03:00:11:11:01 | 2023-02-09 07:53:50 | 13285392653917159195 |         0 |
+-------------------+---------------------+----------------------+-----------+
1 row in set (0.001 sec)
```

with `updated` equals to timestamp of RADIUS Accounting-Request (Start) message.

However, node appears as `offline` in GUI. API returned:

```
			"online": "off",
```

**To Reproduce**
1. Send a RADIUS Accounting-Request (Start) message to PacketFence
2. Check node status in GUI

=> node is offline

**Expected behavior**
node is online

**Additional context**
Setting `process_bandwidth_accounting=enabled` is enabled.

### PR DESCRIPTION
**Pull request by jrouzierinverse**
# Description
Fix scenario where after an accounting stop then start a node's online status is off.
Also fix the node current session cleanup cron job.

# Issue
fixes #7518

# Delete branch after merge
YES

# Checklist
(REQUIRED) - [yes, no or n/a]
- [ ] Document the feature
- [x] Add unit tests
- [ ] Add acceptance tests (TestLink)

### CODE REVIEW

File: go/cron/window_batch_task_test.go [hunk 1]
```diff
@@ -332,3 +332,45 @@ func TestAcctCleanup(t *testing.T) {
...
+		t,
+		"node_current_session_cleanup",
+		map[string]interface{}{
+			"timeout": 10.0,
+			"batch":   100.0,
+			"window":  float64(12 * 60 * 60),
+		},
+		[]string{
+			"DELETE FROM node_current_session",
+			`INSERT INTO node_current_session (mac, updated, last_session_id, is_online) VALUES
```
Comment (satkunas) [inline go/cron/window_batch_task_test.go:347]: Adjust the test, or add another test to assert `updated` and `last_session_id` are overwritten on duplicate key

**Review by satkunas — CHANGES_REQUESTED** [review]
